### ISSUE TITLE
How load pictures from javascript array ?

### ISSUE THREAD

**Issue by visualight** [vvvmax/unitegallery#46]
Hello,

I need to know how iI can load pictures dynamically from a javascript Array or an ajax call (from a url).
Thank you

**Comment by vvvmax** [comment]
Hi. I dont have something ready for this. You can produce gallery html and
then call the gallery js function

**Comment by davidlb** [comment]
Yes, it would be nice to be able to add items to the gallery from API. It would be useful for infinite scroll gallery.

**Comment by vvvmax** [comment]
HI. now the add feature is not exists in the gallery. I'm planning to add
it in the further versions of the gallery, but now it's absent, sorry.

**Comment by rubentlc** [comment]
Hi,

Any news about this thread? I'm developing a mobile app and found this lib very cool! But without enable add photos dynamically it comes unused for me =/.

Thanks

**Comment by forger** [comment]
Yeah, big upvote for this feature from my end too! paging or infinite scrolling would be really neat.
I need to **reload the gallery** after a photo upload and subsequently **select the just uploaded photo**. [This gist](https://gist.github.com/forger/f091bdd572722acb36ce92043f51646c) shows my approach. 
It requires destroying and rebuilding the gallery and is therefore not as smooth an UX as it could be, but the best I could come up with until the API permits modifying the underlying item collection.

**Comment by vvvmax** [comment]
HI, thanks for the suggestions. I'm working now on the admin sorting (in wp
version), then will start working on the ajax add items, and make a
documented replace items option as well.

**Comment by Swaraja** [comment]
Hi ,

any updates on uploading images dynamically using ajax/json .

Regards,
Swaraj

**Comment by Elyahou** [comment]
It would be excellent to be able to add dynamically an image to an existing gallery

**Comment by khoran** [comment]
I would also find this feature very useful. The rest of the library works great.

**Comment by FedericoBenedetti1976** [comment]
big upvote for this feature, my gallery has 100/300 photo each i'm looking how load on infinite scroll by ajax.
Any news?
thank's

**Comment by novusbd** [comment]
This guy stopped supporting the script apparently.

**Comment by avihenri** [comment]
I would also love to do this as trying to implement a lazyload feature but can't dynamically load the images. This seems like the best gallery as well. Wish I had known a week ago or I wouldn't have started with it!
